app/models/tweet.rb

```diff
@@ -1,9 +1,14 @@
 # frozen_string_literal: true
 
 class Tweet < ApplicationRecord
-  validates :user, presence: true
+  validates :user_id, presence: true
   validates :title, presence: true
   validates :content, presence: true
 
   belongs_to :user
+  has_many :favorites
+
+  def favorited_by?(user)
```

Comment:
今の実装だと、favorited_by?メソッドが、
userがidというラッパーメソッドを持つことを知っていないといけないので、user_idを引数に渡すようにしましょう。
（今回の状態はモジュール結合度で言うところの、「スタンプ結合」と言います。）

```suggestion
  def favorited_by?(user_id)
```

Reference: https://zenn.dev/taiga533/articles/e08ad4f4af5577079b5b#%E3%82%B9%E3%82%BF%E3%83%B3%E3%83%97%E7%B5%90%E5%90%88

Reply:
`let(:tweet) { create(:tweet, user: user) }`
テストのときのこれはスタンプ結合にはならない？